MediaBrowser.Server.Mono/OperatingSystemInformationLookup.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace EmbyServer
+{
+    public class OperatingSystemInformationLookup : IOperatingSystemInformationLookup
```

Comment:
Abstract away some calls to `RuntimeInformation`, `Environment`, `Path`. This is needed to be able to unit test behaviour for all platforms without running tests separately on each platform. This allows us to mock what these calls return when testing.